src/battle_ai_script_commands.c

```diff
@@ -1765,15 +1765,15 @@ static void BattleAICmd_if_has_move(void)
     switch (gAIScriptPtr[1])
     {
     case AI_USER:
-        for (i = 0; i < 4; i++)
+        for (i = 0; i < MAX_MON_MOVES; i++)
         {
             if (gBattleMons[sBattler_AI].moves[i] == *movePtr)
                 break;
         }
-        if (i == 4)
+        if (i == MAX_MON_MOVES)
             gAIScriptPtr += 8;
         else
-            gAIScriptPtr = T1_READ_PTR(gAIScriptPtr + 4);
+            gAIScriptPtr = T1_READ_PTR(gAIScriptPtr + MAX_MON_MOVES);
```

Comment:
not correct

Reply:
not even sure how that one got in there, i thought i hadn't done any of these